src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -238,6 +238,16 @@ export class IeObjectsController {
 		res.send(csvContent);
 	}
 
+	@Get('schemaIdentifierLookup/:schemaIdentifier')
+	@ApiOperation({
+		description: 'Get new schemaIdentifier when given the old one as param.',
+	})
+	public async identifierLookup(
+		@Param('schemaIdentifier') schemaIdentifier: string
+	): Promise<string> {
+		return this.ieObjectsService.getRelatedIdentifier(schemaIdentifier);
```

Comment:
we already have related objects as a concept
so i would prefer to not use that naming here
maybe?
```suggestion
		return this.ieObjectsService.getSchemaIdentifierV3(schemaIdentifierV2);
```